beluga_amcl/test/test_amcl_node.cpp

```diff
@@ -536,7 +539,7 @@ TEST_F(TestNode, NoBroadcastWhenInitialPoseInvalid) {
   ASSERT_TRUE(wait_for_initialization());
   ASSERT_FALSE(tester_node_->can_transform("map", "odom"));
   tester_node_->publish_laser_scan();
-  ASSERT_TRUE(wait_for_pose_estimate());
+  ASSERT_FALSE(wait_for_pose_estimate());
```

Comment:
@nahueespinosa hmm, shouldn't this be `ASSERT_TRUE`? Same elsewhere.

Reply:
Could be if we fallback to initializing from a map when the initial pose is invalid, take a look at https://github.com/Ekumen-OS/beluga/pull/328/commits/aa1bbbc7a804c311e7640a74d05b451e16493e8c.